### PR TITLE
Dev Manager role should show in every TOHE lobby :trollface: 

### PR DESCRIPTION
**Pull request by FuroYT**
(This is a small change but cool i think)

### CODE REVIEW

**Comment by NikoCat233** [comment]
This may cause some user's name get blocked or covered

**Comment by FuroYT** [comment]
I tried on an TOHE v1.1.1 build and it works fine didn’t notice any problem

**Comment by NikoCat233** [comment]
Its bad to find suffix got blocked by dev tags . this should be working

**Comment by ryuk201198** [comment]
Long tags can cause mass disconnect issues. If dev tag is added, suffix was blocked to prevent from dc issues from happening.

**Comment by FuroYT** [comment]
> Long tags can cause mass disconnect issues. If dev tag is added, suffix was blocked to prevent dc issues from happening.

I havent experienced any disconnect issue with this and I was playing since 2 weeks ago not any disconnect issue

**Comment by ryuk201198** [comment]
length of the tag was the reason gradients were removed and/or made a setting

**Comment by FuroYT** [comment]
> length of the tag was the reason gradients were removed and/or made a setting

The tag itself doesn’t change but it’s just the way that the dev manager tag is shown that changed here so the length disconnect issue isn’t due to this but the servers or game using too much colors itself

File: Modules/Utils.cs [hunk 1]
```diff
@@ -1862,6 +1858,11 @@ public static void ApplySuffix(PlayerControl player)
                     _ => name
                 };
             }
+            
+            if (!name.Contains('\r') && player.FriendCode.GetDevUser().HasTag())
```
Comment (Tommy-XL) [inline Modules/Utils.cs:1862]: Why `&& !Main.AutoMuteUs.Value`  was removed?

Comment (FuroYT) [reply Modules/Utils.cs:1862]: It’s because I changed the code before it was added

**Review by Tommy-XL — APPROVED** [review]

**Pull request merged**